recipes/pathspec/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pathspec" %}
+{% set version = "0.5.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54a5eab895d89f342b52ba2bffe70930ef9f8d96e398cccf530d21fa0516a873
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pathspec
+  commands:
+    - pytest --pyargs pathspec  # [unix]
+    # skip symlink tests on windows
+    # https://github.com/cpburnz/python-path-specification/commit/d268375a4c6aa
+    - pytest --pyargs pathspec -k "not test_2_1_check_realpath and not test_2_4_recursive_links and not test_2_5_recursive_circular_links"  # [win]
+
+
+about:
+  home: https://github.com/cpburnz/python-path-specification
+  license: MPL-2.0
+  license_file: LICENSE
+  summary: 'Utility library for gitignore style pattern matching of file paths.'
+
+  description: |
+    pathspec is a utility library for pattern matching of file paths. So far
+    this only includes Git's wildmatch pattern matching which itself is derived
+    from Rsync's wildmatch. Git uses wildmatch for its gitignore files.
+  doc_url: https://python-path-specification.readthedocs.io
+  doc_source_url: https://github.com/cpburnz/python-path-specification/blob/master/doc/source/index.rst
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - proinsias
```

Comment:
@proinsias Are you happy to be listed as a maintainer?

Reply:
thanks for being thorough: also have this downstream:
https://github.com/conda-forge/yamllint-feedstock/pull/12#issuecomment-463885519

Reply:
Perfect, LGTM 👍